integration/inspektor-gadget/top_ebpf_test.go

```diff
@@ -70,7 +70,7 @@ func TestTopEbpf(t *testing.T) {
 	}
 
 	if *k8sDistro == K8sDistroGKECOS {
-		t.Skip("Skip running top ebpf gadget on GKE COS: see pull_request #2280")
+		t.Skip("Skip running top ebpf gadget on GKE COS: see pull request #2280")
```

Comment:
Please move these changes to the first commit in the PR. (Currently they're in the `docs: Add GKE support` one).